DataFormats/Reconstruction/include/ReconstructionDataFormats/MatchInfoTOF.h

```diff
@@ -41,6 +41,9 @@ class MatchInfoTOF
   void setChi2(float chi2) { mChi2 = chi2; }
   float getChi2() const { return mChi2; }
 
+  void setHitPattern(std::uint8_t hitPattern) { mHitPattern = hitPattern; }
```

Comment:
@noferini note that I added the mask to the MatchInfo, maybe you had other ideas